test/middleware/spec.back-link.js

```diff
@@ -81,6 +81,8 @@ describe('Back Links', function () {
     });
 
     it('adds the previous step to res.locals.backLink', function () {
+        req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
If the baseUrl is '/referrer' then wouldn't it be better to stub the referrer as something like 'http://example.com/referrer/step2'